src/Compilers/Core/MSBuildTask/DotnetHost.cs

```diff
@@ -66,7 +66,7 @@ private static string PrependFileToArgs(string pathToTool, string commandLineArg
         {
             var builder = new CommandLineBuilderExtension();
             builder.AppendFileNameIfNotNull(pathToTool);
-            builder.AppendTextUnquoted(commandLineArgs);
+            builder.AppendTextUnquoted(" " + commandLineArgs);
```

Comment:
I assume the builder is based on StringBuilder. 
If so, is it better to call AppendTextUnquoted twice (once for the space and once for `commandLineArgs`) than to concat strings?

Reply:
Yep - just updated this PR. https://github.com/Microsoft/msbuild/blob/master/src/Utilities/CommandLineBuilder.cs#L362